LUA/blockControl.lua

```diff
@@ -12,7 +12,7 @@
 
 -- Table of named trains having an optional specific train signal to start/stop this train individually.
 -- (You can omit the "#" as first character.)
-train = {} 		  
```

Comment:
I had used my default settings for indention (tabs with visual length 4) in contrast to your settings (2 spaces).
I now switched to your style and reformatted the code.